www-apps/jekyll/jekyll-4.3.1.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+USE_RUBY="ruby26 ruby27"
```

Comment:
ruby-2.6 doesn't exist in our tree anymore, it can be removed. 

Then again ruby-3.0 seems to be supported by deps of this package, maybe you can add that?